packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -480,6 +480,26 @@
             "name": "stepfunctions_previewstatemachine",
             "description": ""
         },
+        {
+            "name": "workflowStudio_opened",
```

Comment:
I see, I assume they should go here eventually
https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json

Does it work if I first add the metrics one by one here for the ease of development to avoid handling multiple repositories and then move all of them to the common package? I will create a task for it to make sure this is not missed